go.mod

```diff
@@ -17,11 +17,11 @@ require (
 	github.com/spf13/cobra v1.8.0
 	google.golang.org/grpc v1.59.0
 	google.golang.org/protobuf v1.31.0
-	k8s.io/api v0.29.0-rc.1
```

Comment:
Should we wait for csi-lib-utils update to v0.17.0 to include the security fix?

Reply:
I mainly want to get the deps for the client to be updated in this PR. Can we get this merged first and have another update for csi-lib-utils?

Reply:
We can do that too